_includes/about.html

```diff
@@ -3,29 +3,33 @@
         <div class="row">
             <div class="col-lg-8 col-lg-offset-2">
                 <h2>The Concept</h2>
-                <p>It's unformal and community based. You want to give a workshop or a talk? Do it. You want to build a project with others? Do it. This is also a good time for uncoding activities (like sports!) and for knowing each other better. It comes from <a href="http://railscamps.com/">the original RailsCamp</a>.</p>
+                <p>It's non-formal and community based. You want to give a
+                workshop or a talk? Do it. You want to build a project with
+                others? Do it. This is also a good time for non coding activities (like sports!) and for knowing each other better. It comes from <a href="http://railscamps.com/">the original RailsCamp</a>.</p>
 
                 <h2>Code Of Conduct</h2>
 
-                <p>We are dedicated to provide a harassment-free camp for everyone. We do not tolerate harassment and unappropriate behavior. We follow the same <a href="http://2015.arrrrcamp.be/coc/">Code of Conduct than Arrrrcamp</a>.</p>
+                <p>We are dedicated to provide a harassment-free camp for
+                everyone. We do not tolerate harassment and inappropriate  behavior. We follow the same <a href="http://2015.arrrrcamp.be/coc/">Code of Conduct than Arrrrcamp</a>.</p>
```

Comment:
you can refer this one now https://github.com/brug-be/code_of_conduct
as an example of implementation http://fosdem.rubybelgium.be/coc